### DISCUSSION TITLE
Where to find the doc for Starcraft with full data for every unit?

### DISCUSSION THREAD

**Discussion by playgithub** [bwapi/bwapi#893] [Q&A] [answered]
Where to find the doc for Starcraft with full data for every unit?

**Comment by playgithub** [comment, marked as answer]
Found one: http://classic.battle.net/scc

**Comment by dgant** [comment]
Unit data: https://docs.google.com/spreadsheets/d/1jlYDkXW2pQey1oGJlQRBiGjdMs0kfrdHAwzVI2zAjrU/edit#gid=0
Weapon data: https://docs.google.com/spreadsheets/d/1YWFzY0_MEE4fOy9HrvZCYSpRqjgSBdCeRYVxB6zmJpQ/edit#gid=0
Unit type transitions: https://docs.google.com/document/d/1p7Rw4v56blhf5bzhSnFVfgrKviyrapDFHh9J4FNUXM0/edit

**Comment by heinermann** [comment]
https://bwapi.github.io/namespace_b_w_a_p_i_1_1_unit_types.html

**Reply by heinermann** [reply]
Or https://bwapi.github.io/bwapi5/namespace_b_w_a_p_i_1_1_unit_types.html
